server.c

```diff
@@ -93,27 +97,89 @@ int main(int argc, char *argv[])
 	}
 
 	// TO DO: instead of this, implement checking user from file
-	if ((strcmp(login, "maslo") && strcmp(haslo, "qwerty")) == 0)
-	{
-		if (sendall(client_socket, "TAK\n") == -1)
+int i = checklog(client_socket, login,  haslo); // char *login, char *haslo));
```

Comment:
- lack of tab before line
- please change name "i" for something significant
- we need that comment at very end of line?